cli/tests/upgrade.bats

```diff
@@ -35,17 +35,17 @@ project_teardown() {
 assert_old_hello() {
   run jq -r ".packages.\"$NIX_SYSTEM\".hello.input.attrs.narHash" "$LOCK_PATH"
   assert_success
-  assert_output "$OLD_NAR_HASH"
+  assert_output "$PKGDB_NIXPKGS_NAR_HASH_OLD"
 }
 
 assert_new_hello() {
   run jq -r ".packages.\"$NIX_SYSTEM\".hello.input.attrs.narHash" "$LOCK_PATH"
   assert_success
-  assert_output "$NEW_NAR_HASH"
+  assert_output "$PKGDB_NIXPKGS_NAR_HASH_NEW"
 }
 
-OLD_NAR_HASH="sha256-1UGacsv5coICyvAzwuq89v9NsS00Lo8sz22cDHwhnn8="
-NEW_NAR_HASH="sha256-5uA6jKckTf+DCbVBNKsmT5pUT/7Apt5tNdpcbLnPzFI="
+PKGDB_NIXPKGS_NAR_HASH_OLD="sha256-1UGacsv5coICyvAzwuq89v9NsS00Lo8sz22cDHwhnn8="
+PKGDB_NIXPKGS_NAR_HASH_NEW="sha256-5uA6jKckTf+DCbVBNKsmT5pUT/7Apt5tNdpcbLnPzFI="
```

Comment:
question: Are these different vars then set in the suite setup?

Reply:
Forgot to drop them here